### DISCUSSION TITLE
Validation Callsign

### DISCUSSION THREAD

**Discussion by 14FRS711** [richonguzman/LoRa_APRS_iGate#128] [Ideas]
hello, it is a shame to have added the check Valid Callsign :(

**Comment by richonguzman** [comment]
Why is it?

What's your callsign?

**Comment by 14FRS711** [comment]
since the update of 06/21/2024 the source code only accepts callsigne in the format A0AAA, AA0AA, but not AAA000.
even by modifying the code, those who receive cannot redirect my callsigne.
My callsigne : FRS711

**Comment by Mane76** [comment]
Is this a valid ham radio callsign ?

**Comment by SA7SKY** [comment]
Not a callsign. Rather an intruder to APRS-IS. Sorry!

**Comment by 14FRS711** [comment]
no it is not a valid amateur radio callsign. 
aprs.fi authorizes my callsign, even if it is not an amateur radio callsign.
in the terms of use of aprs.fi, it is not obligatory to be licensed.
the use of Lora in my region is open without license. 🙂

**Reply by richonguzman** [reply]
> no it is not a valid amateur radio callsign. aprs.fi authorizes my callsign, even if it is not an amateur radio callsign. in the terms of use of aprs.fi, it is not obligatory to be licensed. the use of Lora in my region is open without license. 🙂

if its not a valid amateur callsign is not a callsign at all.

you can use APRS.FI without a license on certain services like WX stations.

for igate, trackers and digirepeaters there is a callsign needed, why? because the process of validation of sending from LoRa or AFSK/AX.25 needs valid callsigns to work to.

**Comment by 14FRS711** [comment]
if I understood correctly, I cannot use another calling format because it is not compatible with AX25 mode. OK
First question: what does this cause if my calling does not meet the AX25 standard in routing? (in the basic functionality that I can see, it works well, I am heard, I listen and I transmit.

second question: is there the possibility of still assigning other calls that would use another protocol?

Sorry, I'm a novice, I like discovering new technologies and you are of great help.
I invested in this project in my campaign, I am sad to share it with my amateur radio friends.
Thank you

**Reply by richonguzman** [reply]
You should have a valid callsign, even the rookie/novice/noob type (that its usually free after a test) will be enough. This is to asure you know what you are doing with RF signals.

this is a rule for APRS, for Ham Ops (there are non Ham Ops intender LoRa firmwares also)

2) I did not get the "another protocol" question (maybe its the recently answer line)

3) we all start as novice , and its great that you are willing to participate, but there are some rules we all must follow to make this better (and somehow controlled)

sadly most "non" Ham Ops that get into testing and playing do more harm to APRS that positive, so this minor thing is to just avoid minimal harm.

**Comment by SA7SKY** [comment]
Just because the validation generator gives out a number on any kind of term/name/expression  does not mean you are accepted. And by the way: a callsign/validation combination is not accepted by aprs.fi but by the APRS-IS (internet server). aprs.fi is a service provider that generates nice graphical display out of the APRS-IS' data.
Please make a license, read about a system you want to participate in, talk to radio club members for tricks and tips, then buy a radio, then read that manual, then start transmitting.
You are more than welcome then!
